scss/grid/_grid.scss

```diff
@@ -30,6 +30,11 @@ $grid-column-align-edge: true !default;
 /// @type Number
 $block-grid-max: 8 !default;
 
+/// Selector used for an alias of column (with @extend). If `false`, no alias is created.
+/// @since 6.4.0
```

Comment:
I hesitated to put the "@since" because I find that the main problem is that when we do a "pull request" we do not know the version for which it will really be applied.
If the definition of the variable changes (for example in the types) I think it would change the "@since" to signal that it is no longer used as before. So as soon as there is a "@since" it can either be an addition of variable or a variable incompatibility.